aries-site/src/examples/components/page/PageBackGroundExample.js

```diff
@@ -0,0 +1,326 @@
+import React, { useContext } from 'react';
+import {
+  Box,
+  Grid,
+  Heading,
+  Paragraph,
+  Page,
+  PageContent,
+  Text,
+  Button,
+  Image,
+  ThemeContext,
+} from 'grommet';
+
+import { demoStyle } from './demoStyle';
+
+export const PageBackGroundExample = () => (
+  <Page kind="wide" flex="grow" {...demoStyle}>
+    <PageContent
+      background={{
+        fill: 'horizontal',
+        image: 'url(/images/abstract-color-clouds.jpg)',
+        size: 'contain',
+        position: 'left',
+        color: '#FAFBFD', // custom to match image background
+      }}
+      {...demoStyle}
```

Comment:
should we add a comment how the "demoStyle" should be removed in production?

Reply:
done